src/app/components/Search/ResultsList.jsx

```diff
@@ -4,8 +4,7 @@ import {
 } from 'underscore';
 import Instances from './Instances';
 
-const ResultsList = (results) => {
-  console.log(results);
+export const ResultsList = (results) => {
```

Comment:
You can also `export default const ...` so when you import you can do `import ResultsList from '...'` instead of `import { ResultsList } from '...'`.